prime-router/src/main/kotlin/history/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -219,6 +194,48 @@ class DatabaseSubmissionsAccess(private val db: DatabaseAccess = WorkflowEngine.
         }
     }
 
+    /**
+     * Fetch the details of an actions relations (descendents)
+     *
+     * This is done through a recursive query on the report_lineage table
+     */
+    override fun <T> fetchRelatedActions(
+        submissionId: Long,
+        klass: Class<T>
+    ): List<T> {
```

Comment:
kdoc for parameters, to help future developers on this.

Reply:
done!